spec/controllers/badges_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe BadgesController, type: :controller do
+  describe 'GET index' do
+    it 'returns http success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.